data/json/recipes/chem/drugs.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "caffeine",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_DRUGS",
+    "skill_used": "chemistry",
+    "difficulty": 9,
```

Comment:
Difficulty of 9 means this is super obscure recipe with super complex reactions only world-class chemists should be able to conduct. In game terms it's effectively a mutagen-level recipe (or even higher as there are 8-level mutagen recipes). I suggest lowering the difficulty substantially.
Difficulties of two other recipes might require a review too.